test/metric_test.rb

```diff
@@ -26,7 +26,7 @@ def test_name_prefix
 
   def test_handle_bad_tags
     assert_equal ['ignored'], StatsD::Instrument::Metric.normalize_tags(['igno|red'])
-    assert_equal ['lolclass:omglol'], StatsD::Instrument::Metric.normalize_tags({ :"lol::class" => "omg::lol" })
+    assert_equal ['lol::class:omg::lol'], StatsD::Instrument::Metric.normalize_tags({ :"lol::class" => "omg::lol" })
```

Comment:
SO how will this show up in DD? `lol` => `:class:omg::lol`? Or `lol::class` => `omg::lol`?